recovery/recovery.go

```diff
@@ -196,15 +229,15 @@ func Install(version string) error {
 	return nil
 }
 
-func createWritable(keyfile string, keyfileSize int, cryptdev string) error {
-	logger.Noticef("Creating new writable")
+func createWritable(keyBuffer []byte, cryptdev string) error {
+	logger.Noticef("Creating new ubuntu-data")
```

Comment:
```suggestion
	logger.Noticef("Creating new ubuntu-data partition")
```

It would be good to always refer to partitions as `"foo" partition`, because the bare name like ubuntu-data is not associated with partitions in any way.

Reply:
Fixed.